optuna/terminator/erroreval.py

```diff
@@ -27,11 +27,33 @@ def evaluate(
 
 @experimental_class("3.2.0")
 class CrossValidationErrorEvaluator(BaseErrorEvaluator):
+    """An error evaluator for objective functions based on cross-validation.
+
+    This evaluator evaluates objective functions' statistical error, which comes from the
+    randomness of dataset. This evaluator assumes that the objective function is evaluated by
+    cross-validation and uses the scaled variance of the cross-validation scores as the
```

Comment:
```suggestion
    randomness of dataset. This evaluator assumes that the objective function is the average result of a
    cross-validation and uses the scaled variance of the cross-validation scores as the
```

I think we should state clearly what we assume.